earth_enterprise/src/installer/common.sh

```diff
@@ -153,6 +153,13 @@ show_no_tmp_dir_message()
     echo -e "scons release=1 installdir=$1 install\n"
 }
 
+show_corrupt_tmp_dir_message()
+{
+    echo -e "\nThe temp install directory specified [$1] is corrupt (failed on copy)."
+    echo -e "Please specify the path of the extracted install files or first run"
+    echo -e "scons release=1 installdir=$1 install\n"
```

Comment:
Should we add a "-j8" to the 'scons' line to make it consistent with the wiki, also?